agent/command/papertrail_trace_test.go

```diff
@@ -81,7 +81,7 @@ func TestPapertrailTrace(t *testing.T) {
 	}
 
 	ctx := context.Background()
```

Comment:
Can we do this instead?
```go
ctx, cancel := context.WithCancel(context.Background())
```
There's an issue where if there's background work being done (like a goroutine), we have to ensure that they get cleaned up when the context exits. If we use the background context, the background work may never exit and it pollutes other tests by leaving stuff running in unrelated later tests.

Reply:
This context is never cancelled and no goroutines are spawned in the background, so we cannot leak anything

Reply:
This is true, but we could change the implementation in the future, so using the rule above is safer than assuming what the implementation will be like forever. We've had issues in other Go repos (and on rare occasion in Evergreen) where we have to disentangle and fix many tests that don't cancel their context, and the tech debt accumulates fast in Evergreen.

Reply:
Sure, I changed to use the above